.github/workflows/main.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# This is a basic workflow to help you get started with Actions
+
+name: WeeklyOpenApiDocsDownload
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: 
```

Comment:
Is there a possibility that you need a path spec restriction on creating PRs? Probably not because of the specific branch spec.